app/src/main/java/com/plan2meet/dao/DatabaseManager.kt

```diff
@@ -0,0 +1,4 @@
+package com.plan2meet.dao
+
+class DatabaseManager {
```

Comment:
I would recommend creating a database manager class to access DB anywhere in the project.

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.